src/python/gudhi/simplex_tree.pyx

```diff
@@ -17,6 +17,9 @@ __author__ = "Vincent Rouvreau"
 __copyright__ = "Copyright (C) 2016 Inria"
 __license__ = "MIT"
 
+# For unitary tests purpose
+__GUDHI_USE_EIGEN3 = _GUDHI_USE_EIGEN3
```

Comment:
We could also put this in `__init__.py`, since it is already generated by cmake.

Reply:
You are right this is simpler. Done on 957da77